src/styles/constants.scss

```diff
@@ -0,0 +1 @@
+$mobile-max-width: 480px;
```

Comment:
Read [this](https://mattferderer.com/use-sass-variables-in-typescript-and-javascript) and see how you can export all of your scss variables into a JavaScript object.
This will help you along the way and allow you to have a [SSOT](https://edunceputans.medium.com/single-source-of-truth-and-problems-with-implication-in-an-organisation-588883492133)
More important patterns to know about: [DRY, KISS](https://www.linkedin.com/advice/0/how-do-you-simplify-your-code-dry-kiss-principles).